packages/auth/README.md

```diff
@@ -15,6 +15,46 @@ This module can be used as a framework agnostic wrapper using framework specific
 
 *Check back soon for documentation on wallet selector wrapper methods and example usage with other frameworks in the near future.*
 
+
+# config vars
+
+You can set and NEED to set the Near Network you gonna use on your app in three ways:
```

Comment:
Same as above suggestion, do we need to specify this in both objects or can we do this in a single core lib then access from the others?